src/components/Wizard/index.tsx

```diff
@@ -76,7 +77,32 @@ export default function Wizard() {
         template,
       });
 
-    return <NewEditImportPane onNewMessage={handleNewMessage} />;
+    const handleEditMessage = (id: string) => {
```

Comment:
Should use `useCallback(fn, [messages])` where messages comes from the hook i mentioned.